<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailProfilesDlg</class>
 <widget class="QDialog" name="MailProfilesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>437</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mail profiles</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qHostMonitor.qrc">
    <normaloff>:/img/action/mail.png</normaloff>:/img/action/mail.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,3">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lblProfiles">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Profiles</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnNew">
            <property name="text">
             <string>&amp;New</string>
            </property>
            <property name="icon">
             <iconset resource="../qHostMonitor.qrc">
              <normaloff>:/img/new.png</normaloff>:/img/new.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCopy">
            <property name="text">
             <string>&amp;Copy</string>
            </property>
            <property name="icon">
             <iconset resource="../qHostMonitor.qrc">
              <normaloff>:/img/action/copy.png</normaloff>:/img/action/copy.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRename">
            <property name="text">
             <string>&amp;Rename</string>
            </property>
            <property name="icon">
             <iconset resource="../qHostMonitor.qrc">
              <normaloff>:/img/action/properties.png</normaloff>:/img/action/properties.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelete">
            <property name="text">
             <string>&amp;Delete</string>
            </property>
            <property name="icon">
             <iconset resource="../qHostMonitor.qrc">
              <normaloff>:/img/trash.png</normaloff>:/img/trash.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listProfiles">
         <item>
          <property name="text">
           <string>Mail to Boss</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mail to manager</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mail to admin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Page to admin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mail to customer</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0,2">
          <property name="spacing">
           <number>7</number>
          </property>
          <item>
           <widget class="QLabel" name="lblEmailBody">
            <property name="text">
             <string>E-Mail body</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbEmailBody">
            <item>
             <property name="text">
              <string>Text</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HTML</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lblCharset">
            <property name="text">
             <string>Charset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbCharset">
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="currentText">
             <string/>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToMinimumContentsLength</enum>
            </property>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>iso-8859-1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>iso-8859-2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>koi8-r</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>windows-874</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>windows-1250</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>windows-1251</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>windows-1252</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>windows-1253</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>windows-1254</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>windows-1255</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>windows-1256</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>windows-1257</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>windows-1258</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>us-ascii</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="emailEdit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="btnBoxMailProfiles">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qHostMonitor.qrc"/>
  <include location="../qHostMonitor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnBoxMailProfiles</sender>
   <signal>accepted()</signal>
   <receiver>MailProfilesDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBoxMailProfiles</sender>
   <signal>rejected()</signal>
   <receiver>MailProfilesDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
